CLIENT-SPECIFICATION.md

```diff
@@ -39,9 +39,13 @@ Option             | Required?   | Meaning
 `-u`, `--update`   | Conditional | Updates the offline cache of pages. MUST be implemented if caching is supported.
 `-l`, `--list`     | No          | Lists all the pages in the current platform to the standard output.
 `-L`, `--language` | No          | Specifies the preferred language for the page returned. Overrides other language detection mechanisms. See the [language section](#language) for more information.
+`-S`, `--shortform`| No          | If set, will filter {{[*|*]}} patterns in examples to show their shortform option (first element).
+`-V`, `--longform` | No          | If set, will filter {{[*|*]}} patterns in examples to show their longform option (second element).
 
 Clients MUST implement both the short and long versions of an option.
```

Comment:
Does this need clarification? The MUST appearing right after `--shortform` and `--longform` options might cause confusion.

Reply:
Yes, it is important that clients support both the short and the long forms of all arguments.

It is a MUST to ensure consistency of CLI experience ref `tldr tldr` etc. I strongly advise that it is left as a MUST.

Any particular reason for `-V` here as the short form of `--longform`?

Reply:
> Yes, it is important that clients support both the short and the long forms of all arguments.

You misunderstand me. What will happen when someone reads the specs, reads about shortform/longform options that are optional according to the table and then immediately after reads they you "MUST implement both the short and long versions of an option"? I would assume confusion.
I was asking if the wording needs clarification so that it's apparent that it's talking about the shortform/longform pair and not `--shortform` and `--longform`

> Any particular reason for -V here as the short form of --longform?

`-l` and `-L` were taken. I went for the next best option `-V`  short for verbose.

Reply:
Hmmm, I see your reasoning.

I think the wording "of an option" us unchanged from the time I originally wrote the client spec, though perhaps it would be clearer to say "of each given argument" instead?

Reply:
Also ref `-V`, I can't help but think that could cause confusion. What about `-e` for expanded form, or even completely different `-a` and `-A` for short and long forms of arguments respectively?

Reply:
`-o` for shortform and `-O` for longform. Short for "option"?

Reply:
I moved the long+short option requirement to the start. I assume that is much clearer with its intentions.